src/repositories/LastestItemStore.ts

```diff
@@ -45,7 +45,7 @@ export class LastestItemStore<K extends number, T> {
   }
 
   public receipt(id: K) {
-    const version = this.store.getVersionOfPreviousWrite(id);
+    const version = this.store.getVersionOfPreviousWrite(id) || 0;
```

Comment:
Is it really OK to set version=0?
I feel like we should return "not found" instead, but not sure

Reply:
Added a test for undefined